datumaro/plugins/vgg_face2_format.py

```diff
@@ -23,20 +23,29 @@ class VggFace2Path:
     IMAGES_DIR_NO_LABEL = 'no_label'
 
 class VggFace2Extractor(SourceExtractor):
```

Comment:
If extractor outputs multiple subsets, it must be inherited from `Extractor`.

Reply:
Fixed.